designer/app.py

```diff
@@ -597,7 +597,7 @@ def _fbrowser_load(self, instance):
         except:
             error = 'Cannot load empty file type'
     
-        self.statusbar.show_message(error)
+            self.statusbar.show_message(error)
```

Comment:
This won't show the error if it is set in the `else:` block. It would be better to set `error = None` at the beginning, and then do `if error: self.statusbar.show_message(error)`.